keyvalue/clients/src/main/java/tachyon/client/keyvalue/KeyValueStoreWriter.java

```diff
@@ -48,4 +51,15 @@
    */
   void put(ByteBuffer key, ByteBuffer value) throws IOException, TachyonException;
 
+  /**
+   * Merges a key-value store to the current store and deletes the merged store afterwards.
+   *
+   * @param uri URI of the key-value store to be merged
+   * @throws IOException if non-Tachyon error occurs
+   * @throws FileDoesNotExistException if the uri does not exist
```

Comment:
I think that's because any path from the view of `FileSystem` can only be a file, so there is no case that a URI exists but is not a file.